### PR TITLE
Compliance

### PR DESCRIPTION
**Pull request by nkolba**
Adding page explaining the terms of FDC3 compliance

### CODE REVIEW

File: docs/fdc3-compliance.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+>   and sparingly.  In particular, they MUST only be used where it is
+>   actually required for interoperation or to limit behavior which has
+>   potential for causing harm (e.g., limiting retransmisssions)  For
+>   example, they must not be used to try to impose a particular method
+>  on implementors where the method is not required for
+>   interoperability.
+
+These rules would apply only to standards work within FDC3.  Today, this covers API, App Directory, Context Data, and Intents working groups, it does not apply to the Use Cases working group. 
+
+### Personas:
```
Comment (rikoe) [inline docs/fdc3-compliance.md]: If you use level 3 headings as the highest level they will not show up in the navigation helper. Use level 2 headings instead.

File: docs/fdc3-compliance.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+FDC3 implementors generally fall into 2 categories: platform providers, and application providers.  
+A platform provider provides an implementation of the FDC3 API for applications to use. Implicitly, it connects to one or more App Directories.
+
+An application provider is largely a downstream consumer of FDC3 standards.  It MAY use the API, it MAY use Context Data, it MAY use Intents.  Application providers are only required to comply with the standards they make use of.  
+
+Depending on persona, implementation compliance with FDC3 will mean different things.  
+
+#### Platform Provider
```
Comment (rikoe) [inline docs/fdc3-compliance.md]: Make this a level 3 heading which will then show as a sub-heading in the navigation guide.

File: docs/fdc3-compliance.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+To meet the requirements of FDC3, a platform provider would need to support the following:
+
+* Connection to 1 or more App Directories meeting the FDC3 App Directory standards
+* Providing an API to all applications running in the context of the platform that meets the FDC3 API standards, including:
+    * Support of FDC3 Context Data and Intents standards
+    * Support for Intent and Context resolution using a resolver UI
+* At least one workflow of one of the use cases marked as Accepted by the Use Cases working group SHOULD be satisfied by the implementation.
+
+#### Application Provider
```
Comment (rikoe) [inline docs/fdc3-compliance.md]: Level 3 heading.

File: docs/fdc3-compliance.md (hunk 1 above)
Comment (nkolba) [reply docs/fdc3-compliance.md]: nice catch.  corrected

**Pull request merged**